### ISSUE TITLE
#question: It there any known issues on ZSH?

### ISSUE THREAD

**Issue by roalcantara** [QuietImCoding/script-kitty#1]
Firstly, thank you so much for your sharing this script.

**TLDR:**

Is there known any issues for installing it on `ZSH` environments?

**The Long reading:**

My question is rather an n00b; for that, I apologize for upfront.

I'm really rooky in dealing with shell scripts, and I could not find any instructions for installing such a script on `ZSH` environments. At first, I reckon that it would not be any issues, and I'm giving a try anyway.

However, as my lack of knowledge on shell scripts in general, I figured that it could not harm asking if you happen to know any issues using it on `ZSH environments`. I would greatly appreciate not "waste my time" trying in that case. 😅

If you don't happen to know how it works on `ZSH` or if you are not keen on it at all, please, discard this message and feel free to close the issue.

 I'll haply figure out, eventually. Hopefully!  🤞🏼

Cheers.

**Comment by roalcantara** [comment]
#first!

**Comment by roalcantara** [comment]
> #first!

I always wanted to that. Could not resist!. I beg your pardon! 🙏🏼
